app/services/healthcheck.rb

```diff
@@ -1,30 +1,22 @@
-class Healthcheck
+module Healthcheck
```

Comment:
Given its public interface is the `report` method which is always interacting with the cache, how about renaming this to `CachedHealthcheck`?